core/src/main/resources/application.yml

```diff
@@ -94,6 +94,10 @@ feast:
       provider: none
       options:
         basePath: http://localhost:3000
+        subject: email
+        roles:
+          - role:feast:admin
+          - role:feast:{PROJECT}-member
```

Comment:
I am a bit wary about using complex objects as options. I would strongly prefer to stick to K/V here, so perhaps we can turn roles into a CSV.